CPPLINT.cfg

```diff
@@ -1,2 +1,2 @@
 set noparent
-filter=-runtime/reference,-runtime/string,-build/c++11,-build/include_subdir,-whitespace/forcolon,-build/namespaces_literals,-readability/check
+filter=-runtime/reference,-runtime/string,-build/c++11,-build/include_subdir,-whitespace/forcolon,-build/namespaces_literals,-readability/check,-whitespace/braces
```

Comment:
This will disable this whitespace/braces checks everywhere, we could fix that false positive in a much smaller scope by leaving a comment // NOLINT(whitespace/braces) at the of the PutUDPTests.cpp:134 line

Reply:
I found one more occurrence in TailFileTests.cpp:589. It's still not common, so I could change it, but a linter rule with many false positives is not very useful in my opinion.

Reply:
Yeah, I agree if the linter rule is unreliable than its useless. Since this is not just a one-off, I am fine either way.

Reply:
I'll push a change to NOLINT in the next batch of changes, hopefully this week.